build.gradle

```diff
@@ -76,9 +76,19 @@ dependencies {
 	compileOnly fileTree(dir: './JavaCommons-core/build/libs', include: ['*-all.jar']) 
 	testImplementation fileTree(dir: './JavaCommons-core/build/libs', include: ['*-all.jar']) 
 
+
 	// This dependencies is exported to consumers, that is to say found on their compile classpath.
 	//-------------------------------------------------------------------------------------------------------------------
 
+	// iText 
+	api files("./lib/iText.jar")
+
+	//batik SVG
+	api fileTree(dir: './lib/batik/', include: ['*.jar']) 
+
+	// https://mvnrepository.com/artifact/org.w3c.dom/svg
+	implementation 'org.kie.modules:org-apache-batik:6.1.0.Beta1'
```

Comment:
If your adding bartik library here, why are u commiting the jars as well? 

Please remove the uneeded bartik jars if this is used instead.

Reply:
switched to use mvnrepo batik libs ( https://github.com/picoded/JavaCommons-pdf/pull/1/commits/532e9ab83797a7a512f9c5f87f17e1b5b50182b2 ) and remove unused ( https://github.com/picoded/JavaCommons-pdf/pull/1/commits/d4954e2d10ee442c7f4a3bc3a430dd13400e966a )